### ISSUE TITLE
download playlist

### ISSUE THREAD

**Issue by atarbig** [CupCakeArmy/deezer#1]
need option 'download all' playlist

**Comment by CupCakeArmy** [comment]
Yes, I agree, was already implemented but I took it away. I'll implement it with a delay timer, so it's not too obvious for deezer
